check_graphite

```diff
@@ -31,20 +33,28 @@ EXIT_WARNING    = 1
 EXIT_CRITICAL   = 2
 EXIT_UNKNOWN    = 3
 
+requests.packages.urllib3.disable_warnings()
+
 def check(host, metric, warning_threshold, critical_threshold,
-          invert, function, duration, user, password):
+          invert, function, duration, user, password, grafana, datasourceid, authkey):
 
-    if not host.startswith('http'):
-        host = 'http://%s' % host
+    if not host.startswith('https'):
```

Comment:
since this is a breaking change, I'd prefer if we leave the default as http and let users type out "https://" in the host if they want to use https. the `host.startswith('http')` check will allow `https`, but changing the check to `startswith('https')` would prepend `https://http://` if the host already starts with `http://`.